src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -253,13 +252,7 @@ private function showEntries($type, Request $request, $page)
         $entries = new Pagerfanta($pagerAdapter);
 
         $entries->setMaxPerPage($this->getUser()->getConfig()->getItemsPerPage());
-        try {
-            $entries->setCurrentPage($page);
-        } catch (OutOfRangeCurrentPageException $e) {
-            if ($page > 1) {
-                return $this->redirect($this->generateUrl($type, array('page' => $entries->getNbPages())), 302);
-            }
-        }
```

Comment:
You're right. 

I thought that our previous behavior was to avoid an Exception. But no, it was really to redirect (#1551). 
Will fix that.